CMakeLists.txt

```diff
@@ -350,8 +350,8 @@ endif()
 
 add_subdirectory(framework)
 add_subdirectory(include)
-add_subdirectory(core)
 add_subdirectory(drivers)
+add_subdirectory(core)
```

Comment:
`core` needs to be after `drivers` because `TF_PSA_CRYPTO_DRIVERS` is defined in `drivers/CMakeLists.txt` and used in `drivers/CMakeLists.txt`, right? This could use a comment. Someone might want to do the opposite move because they would like to define a variable in `core` and use it in drivers.